tools/continuous_batching/accuracy/continuous_batching_speculative_decoding.cpp

```diff
@@ -20,20 +20,23 @@ std::vector<ov::genai::GenerationConfig> get_spec_decoding_generation_config_exa
         generation_config_greedy_constant.num_assistant_tokens = 5;
     }
 
-    ov::genai::GenerationConfig generation_config_multinomial_constant = ov::genai::multinomial();
```

Comment:
Is this change expected for fast draft speculative decoding as well? Will Eagle3 support multinomial? If yes, let's create a TODO and add ticket number.